code/controllers/subsystem/shuttle.dm

```diff
@@ -216,6 +219,95 @@ SUBSYSTEM_DEF(shuttle)
 	log_game("[key_name(user)] has called the shuttle.")
 	message_admins("[key_name_admin(user)] has called the shuttle.")
 
+	if(legitimacy_check)
+		check_call_legitimacy()
+
+
+/datum/controller/subsystem/shuttle/proc/check_call_legitimacy()
+	if(station_in_trouble())
+		message_admins("Shuttle call is legitimate by the listed metrics.")
+		return
+
+	var/time_to_recall = rand(600, 1500)
+	recall_timer_id = addtimer(CALLBACK(src, .proc/recall_illegitimate, emergency.timer), time_to_recall, TIMER_STOPPABLE)
+	annoyed_admiral_message = pick("Do you know how expensive these stations are?","Stop wasting my time.","I was sleeping, thanks a lot.","Stand and fight you cowards!","You knew the risks coming in.","Stop being paranoid.","Whatever's broken just build a new one.","No.", "<i>null</i>","<i>Error: No comment given.</i>")
```

Comment:
yes, and you null it after use, so if an admin has pre-prepared one, don't override it!